<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>mergeFeaturesDialog</class>
 <widget class="QDialog" name="mergeFeaturesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Merge Features</string>
  </property>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>513</x>
     <y>358</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text" >
    <string>Run!</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>427</x>
     <y>358</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton" >
   <property name="geometry" >
    <rect>
     <x>15</x>
     <y>358</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="polygonToPolyLineGroup" >
   <property name="geometry" >
    <rect>
     <x>15</x>
     <y>10</y>
     <width>580</width>
     <height>220</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string/>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="Q3Table" name="inputOutputTable" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>510</width>
      <height>200</height>
     </rect>
    </property>
    <property name="lineWidth" >
     <number>2</number>
    </property>
    <property name="midLineWidth" >
     <number>0</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="resizePolicy" >
     <enum>Q3ScrollView::Default</enum>
    </property>
    <property name="dragAutoScroll" >
     <bool>false</bool>
    </property>
    <property name="numRows" >
     <number>0</number>
    </property>
    <property name="numCols" >
     <number>1</number>
    </property>
    <property name="showGrid" >
     <bool>true</bool>
    </property>
    <property name="rowMovingEnabled" >
     <bool>false</bool>
    </property>
    <property name="columnMovingEnabled" >
     <bool>false</bool>
    </property>
    <property name="focusStyle" >
     <enum>Q3Table::SpreadSheet</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addFileButton" >
    <property name="geometry" >
     <rect>
      <x>530</x>
      <y>10</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>addFile</string>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset>./icons/Plus32.png</iconset>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFileButton" >
    <property name="geometry" >
     <rect>
      <x>530</x>
      <y>50</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>removeFile</string>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset>./icons/Minus32.png</iconset>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllButton" >
    <property name="geometry" >
     <rect>
      <x>530</x>
      <y>90</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>remove All Files</string>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="icon" >
     <iconset>./icons/Delete32.png</iconset>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3ButtonGroup" name="mergeFeaturesGroup" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>235</y>
     <width>581</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="outputMergedFileButton" >
    <property name="geometry" >
     <rect>
      <x>478</x>
      <y>14</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text" >
     <string>Browse...</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outputMergedFileLabel" >
    <property name="geometry" >
     <rect>
      <x>21</x>
      <y>2</y>
      <width>144</width>
      <height>58</height>
     </rect>
    </property>
    <property name="text" >
     <string>Output Merged File</string>
    </property>
   </widget>
   <widget class="QLabel" name="mergeFeaturesClusterToleranceLabel" >
    <property name="geometry" >
     <rect>
      <x>21</x>
      <y>47</y>
      <width>132</width>
      <height>58</height>
     </rect>
    </property>
    <property name="text" >
     <string>Cluster Tolerance</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="clusterToleranceLineEdit" >
    <property name="geometry" >
     <rect>
      <x>169</x>
      <y>61</y>
      <width>130</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputMergedFileLineEdit" >
    <property name="geometry" >
     <rect>
      <x>171</x>
      <y>15</y>
      <width>301</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="messageLog" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>407</y>
     <width>581</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends></extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends></extends>
   <header>q3table.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>clusterToleranceLineEdit</tabstop>
  <tabstop>outputMergedFileLineEdit</tabstop>
  <tabstop>outputMergedFileButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>helpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
